mautic/mautic
# Add the real field types to the Update lead event

Caution, this PR is not ready for testing. I need a hand with it though.

## Description
**Update lead** campaign event display all lead fields as a text field. Issue reported at https://github.com/mautic/mautic/issues/1556

## Steps to reproduce
1. Make sure you have for example a select lead field. 
2. Edit a campaign, add the Update lead action.
3. The select field is displayed as a lead field.

## Steps to test
Apply this PR and you should see the real field types of the lead fields in the Update lead action.

Thanks to @alanhartless for the suggestions when I get stuck!
